javatests/arcs/core/storage/ReferenceModeStoreDatabaseIntegrationTest.kt

```diff
@@ -222,6 +222,45 @@ class ReferenceModeStoreDatabaseIntegrationTest {
         assertThat(storedBob.collections).isEqualTo(bobEntity.data.collections)
     }
 
+    @Test
+    fun removeOpClearsBackingEntity() = runBlockingTest {
+        val activeStore = createReferenceModeStore()
+        val actor = activeStore.crdtKey
+        val bob = createPersonEntity("an-id", "bob", 42)
+
+        // Add Bob to collection.
+        val addOp = RefModeStoreOp.SetAdd(actor, VersionMap(actor to 1), bob)
+        assertThat(
+            activeStore.onProxyMessage(ProxyMessage.Operations(listOf(addOp), id = 1))
+        ).isTrue()
+        // Bob was added to the backing store.
+        val storedBob = activeStore.backingStore.getLocalData("an-id") as CrdtEntity.Data
+        assertThat(storedBob.toRawEntity("an-id")).isEqualTo(bob)
```

Comment:
Nit: the ordering for assertThat is meant to be `assertThat(actual).isEqualTo(expected)`, which - in this case, makes me think `bob` should be within the `assertThat` call, and storedBob would be in the `isEqualTo` call.